### PR TITLE
South Africa-Lawrence-Baatjies-HomeWorkFixError.md

### PR DESCRIPTION
**Pull request by LawrenceBaatjies**
Errors fixed

# Your Details

- Your Name: Lawrence Baatjies
- Your City: Cape Town- South Africa
- Your Slack Name: Lawrence

# Homework Details

- Module: Leeson 1
- Week: 2

### CODE REVIEW

**Comment by KudahShambare** [comment]
Hey Lawrence, consider changing the wording for your HTML definition, there is no space in between Hyper and text.
I think I'm done with my review

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
-A programming language used on the internet
+JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript specification.A programming language used on the internet.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1.Helpes us to share information.
+2.Enables people to make thier own changes to that information,and share it back.
```
Comment (RyanS-Cpt) [inline homework.md:78]: Hi Lawrence, small typo here thier = their

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-It lets you see into the future
+git log lists the commits made in that repository in reverse chronological order.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+When you made a change on GitHub, it not only saved the change and committed the file at the same time.
```
Comment (RyanS-Cpt) [inline homework.md:130]: Hi Lawrence, I just want to confirm, do you mean that when you push  - it saves changes and commits? maybe we should just check the steps that need to be taken before you "push"

File: homework.md [hunk 3]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A place for us to store and share our code.
+GitHub, Inc. is a subsidiary of Microsoft which provides hosting for software development and version control using Git. It offers the distributed version control and source code management functionality of Git, plus its own features.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+If you have made changes to the same file in both your local copy(on your harddrive) of a Git repository and a remote repository, you'll encounter the “commit your changes or stash them before you can merge.
```
Comment (RyanS-Cpt) [inline homework.md:141]: A simpler understanding may be that it is how you save your work before "pushing" it - would you agree?

File: homework.md [hunk 4]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-Microsoft Word
+Visual Studio Code Editor.
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+ The commit command is used to save changes to a local repository after staging in Git.
```
Comment (RyanS-Cpt) [inline homework.md:153]: I think you may have misread the question Lawrence.

**Review by RyanS-Cpt — COMMENTED** [review]
Just a few observations

**Comment by RyanS-Cpt** [comment]
I have finished my review

File: homework.md (hunk 4 above)
Comment (LawrenceBaatjies) [reply homework.md:153]: thank you Ryan. I've read the question again and made changes to my answer.

File: homework.md (hunk 3 above)
Comment (LawrenceBaatjies) [reply homework.md:141]: Noted. I've simplified my answer.

File: homework.md (hunk 2 above)
Comment (LawrenceBaatjies) [reply homework.md:130]: Noted. I've checked the steps

File: homework.md (hunk 1 above)
Comment (LawrenceBaatjies) [reply homework.md:78]: Noted. rectified.

**Pull request closed without merging**